docs/faq/modeling.md

```diff
@@ -618,6 +618,6 @@ Depending on the answer, LinkML has different modeling constructs to help you, i
 
 - [range](https://w3id.org/linkml/range) constraints, which can refer to classes
 - the ability to assign a slot as an [identifier](https://w3id.org/linkml/identifier), allowing other objects to link to it
-- [inlining](https://w3id.org/linkml/inlining) which determines how relationships are serialized in formats like JSON
+- [inlining](https://linkml.io/linkml/schemas/inlining.html), which determines how relationships are serialized in formats like JSON
```

Comment:
Is this the best way to link out the document file, or is this:

```md
[inlining](../schemas/inlining.md)
```

Reply:
well, earlier in the same document it spells out the URL as i did. i'm fine to change it to a relative path, which might be more sustainable, but then we should do that consistently with the other links in the document.

Reply:
oh, actually, i think that should be
> [inlining](../schemas/inlining)

(no .md)

Reply:
@sujaypatil96 lmk what you want me to do, or feel free to edit this PR! i am not sure why it's failing tests.

Reply:
Tests are not going through because of a recent PR/merge introducing `pyshacl` into the framework.